{"uuid": "d2b601d2-6cc6-432e-877f-f767166436f3", "children": ["cd643477-31be-42e6-9f1b-08454fc7bdf1"], "befores": [{"name": "handler", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable\n", "trace": "  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\Administrator\\PycharmProjects\\po_jrj\\cases\\conftest.py\", line 9, in handler\n    driver = webdriver.chrome()  #调取chrom浏览器\n"}, "start": 1609333895655, "stop": 1609333895655}], "start": 1609333895655, "stop": 1609333895755}